lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -248,15 +248,15 @@ sub login {
         template => 'confirm_operation',
         format => 'HTML',
     );
-    $template->render_to_psgi($request);
+    return $template->render_to_psgi($request);
 }
 
 =item sanity_checks
 Checks for common setup issues and errors if admin tasks cannot be completed/
 
 =cut
 
-sub sanity_checks {
+sub sanity_checks { ## no critic ( RequireFinalReturn )
```

Comment:
I can see that we want to return 'undef' (or a simple  final `return;`) here: this seems to be more of a `procedure` (Pascal terms: not meant to have a return value) than a `function` (explicit return value)